### DISCUSSION TITLE
Code in the Wind #2 [showdown.space]

### DISCUSSION THREAD

**Discussion by rayriffy** [creatorsgarten/discussions#3] [Event ideas]
I think we should hold another Code in the Wind 2, but this time maybe we can make the system to be compete as a co-op (as a group maximum of 2 people).

Last time we held an even around May, it should be good time slot to be held around November.

Organizer notes
---

- Held event in November must be carefully choose date on calendar to not being conflict with collage exam.
- With an idea of co-op competition, we might need a harder challenges for contestants to solve when comparing to the first event.
  - There are many unused challenge from Code in the Wind #1 (and they are quite hard) so should not be a problem.

**Comment by dtinth** [comment]
- เรื่องโจทย์ ตอนจัดรอบแรกเตรียมไว้เยอะ เหลือโจทย์อยู่ในคลังเยอะพอสมควร (โดยเฉพาะพวกที่ยากๆ)

- เลยคิดว่าโหมด Co-Op น่าสนมาก ทีนี้เรื่อง Code Editor จะทำยังไง

   - ถ้าจัดงานแบบ Offline ก็คือสองคนรุมคอมเครื่องเดียวกัน อันนี้ใช้ Editor ปกติได้เลย
   - แต่ถ้าจัดแบบ Online หรืออยากให้สองคนแก้โค้ดพร้อมๆ กันได้ อาจจะต้องทำให้ Editor สามารถ Collaborate แบบ Realtime ได้ด้วย (อาจจะใช้ [Yjs Monaco binding](https://docs.yjs.dev/ecosystem/editor-bindings/monaco) ได้) ถ้ามีคนช่วยทำตรงนี้น่าจะจัดงานได้ไม่ยาก
